content/_code-samples/freeze/py/check_global_freeze.py

```diff
@@ -0,0 +1,45 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountInfo
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnetwork
+
+
+ACCOUNT_ROOT_LEDGER_FLAGS: dict[str, int] = {
+        "lsfPasswordSpent": 0x00010000,
+        "lsfRequireDestTag": 0x00020000,
+        "lsfRequireAuth": 0x00040000,
+        "lsfRequireAuth": 0x00040000,
+        "lsfDisableMaster": 0x00100000,
+        "lsfNoFreeze": 0x00200000,
+        "lsfGlobalFreeze": 0x00400000,
+        "lsfDefaultRipple": 0x00800000,
+        "lsfDepositAuth": 0x01000000,
+    }
+
+def parse_account_root_flags(flags: int) -> list[str]:
+    flags_enabled = []
+    for flag in ACCOUNT_ROOT_LEDGER_FLAGS:
+        check_flag = ACCOUNT_ROOT_LEDGER_FLAGS[flag]
+        if check_flag & flags == check_flag:
+            flags_enabled.append(flag)
+    return flags_enabled
```

Comment:
This is a large pre-amble which is mostly not needed in this snippet - I'd recommend just making a few variables `lsfGlobalFreeze = 0x00400000` and `lsfNoFreeze = 0x00200000` to avoid overwhelming new readers with information they don't need to understand for this specific example. (Same to the other snippets)